pytest_metadata/plugin.py

```diff
@@ -5,7 +5,7 @@
 import os
 import platform
 
-import _pytest._pluggy as pluggy
+import pluggy
```

Comment:
I was just going to make this change myself.
Maybe we should make it a conditional so that we don't break users of the plugin.
```
try:
    import pluggy
except ImportError:
   import _pytest._pluggy as pluggy
```

Reply:
it should be the other way around, else it reports the wrong pytest pluggy version
for completeness it should probably report both, pytest pluggy and global pluggy